Unable to compile with postgres feature

If you try to compile the programs feature branch with `--features postgres` you get

```
Compiling repository v0.1.0 (/Users/m2/GitHub/open-msupply/server/repository)
error[E0433]: failed to resolve: use of undeclared crate or module `document_head_dsl`
   --> repository/src/db_diesel/document.rs:153:29
    |
153 |         diesel::insert_into(document_head_dsl::document_head)
    |                             ^^^^^^^^^^^^^^^^^ use of undeclared crate or module `document_head_dsl`
    |
help: there is a crate or module with a similar name
    |
153 |         diesel::insert_into(document_head::document_head)
    |                             ~~~~~~~~~~~~~

error[E0433]: failed to resolve: use of undeclared crate or module `document_head_dsl`
   --> repository/src/db_diesel/document.rs:155:26
    |
155 |             .on_conflict(document_head_dsl::id)
    |                          ^^^^^^^^^^^^^^^^^ use of undeclared crate or module `document_head_dsl`
    |
help: there is a crate or module with a similar name
    |
155 |             .on_conflict(document_head::id)
    |                          ~~~~~~~~~~~~~

error[E0433]: failed to resolve: use of undeclared crate or module `json_schema_dsl`
  --> repository/src/db_diesel/json_schema.rs:65:29
   |
65 |         diesel::insert_into(json_schema_dsl::json_schema)
   |                             ^^^^^^^^^^^^^^^ use of undeclared crate or module `json_schema_dsl`
   |
help: there is a crate or module with a similar name
   |
65 |         diesel::insert_into(json_schema::json_schema)
   |                             ~~~~~~~~~~~

error[E0433]: failed to resolve: use of undeclared crate or module `json_schema_dsl`
  --> repository/src/db_diesel/json_schema.rs:67:26
   |
67 |             .on_conflict(json_schema_dsl::id)
   |                          ^^^^^^^^^^^^^^^ use of undeclared crate or module `json_schema_dsl`
   |
help: there is a crate or module with a similar name
   |
67 |             .on_conflict(json_schema::id)
   |                          ~~~~~~~~~~~

For more information about this error, try `rustc --explain E0433`.
error: could not compile `repository` due to 4 previous errors
```

--- a/server/cli/src/refresh_dates.rs
+++ b/server/cli/src/refresh_dates.rs
@@ -30,6 +30,7 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("requisition_line", "snapshot_datetime"),
         ("stocktake", "created_datetime"),
         ("stocktake", "finalised_datetime"),
+        ("document", "timestamp"),
     ]
     .iter()
     .map(|(table_name, field_name)| TableAndFieldName {

--- a/server/repository/src/db_diesel/document.rs
+++ b/server/repository/src/db_diesel/document.rs
@@ -150,9 +150,9 @@ impl<'a> DocumentRepository<'a> {
             name: doc.name.to_owned(),
             head: doc.id.to_owned(),
         };
-        diesel::insert_into(document_head_dsl::document_head)
+        diesel::insert_into(document_head::dsl::document_head)
             .values(&row)
-            .on_conflict(document_head_dsl::id)
+            .on_conflict(document_head::dsl::id)
             .do_update()
             .set(&row)
             .execute(&self.connection.connection)?;

--- a/server/repository/src/db_diesel/json_schema.rs
+++ b/server/repository/src/db_diesel/json_schema.rs
@@ -62,9 +62,9 @@ impl<'a> JsonSchemaRepository<'a> {
     #[cfg(feature = "postgres")]
     pub fn upsert_one(&self, schema: &JSONSchema) -> Result<(), RepositoryError> {
         let row = row_from_schema(schema)?;
-        diesel::insert_into(json_schema_dsl::json_schema)
+        diesel::insert_into(json_schema::dsl::json_schema)
             .values(&row)
-            .on_conflict(json_schema_dsl::id)
+            .on_conflict(json_schema::dsl::id)
             .do_update()
             .set(&row)
             .execute(&self.connection.connection)?;

